src/features/continent/Continent.js

```diff
@@ -9,15 +9,31 @@ import europe from './europe.png';
 const Countries = () => {
   const dispatch = useDispatch();
   const { items, totalConfirmed } = useSelector(state => state.continent);
+  const [searchList, setSearch] = useState([]);
+  const [isSearching, setSearching] = useState(false);
 
   useEffect(
     () => {
       if (!items.length) dispatch(fetchCountries('europe'));
     },
-    [],
+    [searchList],
   );
 
+  const handleSearch = searchString => {
+    if (!searchString.length) {
+      setSearch([]);
+      setSearching(false);
+    } else {
+      setSearching(true);
+      const curr = items.filter(v => v.country[0].toLowerCase() === searchString[0].toLowerCase());
+      if (searchList !== curr) {
+        setSearch([...curr]);
+      }
+    }
+  };
+
   const list = items.sort((a, b) => b.confirmed - a.confirmed);
+  const currList = isSearching ? searchList : list;
```

Comment:
- [ ] OPTIONAL
Great job for adding a search filter. However, you can improve the user interface by adding a search icon or a placeholder text to improve the accessibility of your form.
![image](https://user-images.githubusercontent.com/12745474/142254384-3dac1aa8-815c-4d1b-a3cd-45d9cd41c13f.png)